app/api/validations/user.py

```diff
@@ -242,3 +242,35 @@ def validate_new_password(data):
         return is_valid[1]
 
     return {}
+
+
+def validate_forgot_password_email(data):
+
+    email = data["email"]
+
+    if not is_email_valid(email):
+        return messages.EMAIL_INPUT_BY_USER_IS_INVALID
+
+    return {}
```

Comment:
And I would refactor this to


```suggestion
    return {} if is_email_valid(email) else messages.EMAIL_INPUT_BY_USER_IS_INVALID
```

Reply:
Every other possible responses are following this format so, I don't think that we need to make it different from others, can you @isabelcosta please suggest here what to use?

Reply:
I am open for better ways to do this :) So I guess you could try this out :)

Reply:
@isabelcosta I will try this format but just want to know that shouldn't we then need to format the entire code in each & every other file? so, do we again need to create an issue for formatting?